servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/CapturedContextProvider.java

```diff
@@ -17,53 +17,68 @@
 
 /**
  * Functionality related to capturing thread-local like context for later restoration across async boundaries.
+ *
```

Comment:
While Intellij IDEA renders it better, in javadoc HTML website it always requires `<p>` tag to separate paragraphs or the first line. 
```suggestion
 * <p>
```